### ISSUE TITLE
BertForNextSentencePrediction labels

### ISSUE THREAD

**Issue by ArthurCamara** [huggingface/pytorch-transformers#836]
Hi everyone!
I was reading trough the documentation, and, according to https://huggingface.co/pytorch-transformers/model_doc/bert.html#bertforsequenceclassification, it expects that `next_sentence_label` is `1` if B is **not** a next sequence for A, and `0` if B **is** a sequence for B. 
That's somewhat counterintuitive, since most of the datasets (I believe) will assume this problem to be a binary classification. 

Is my assumption correct? Should I flip my dataset before fine-tuning the model?

**Comment by thomwolf** [comment]
You should supply your own labels when using the `BertForSequenceClassification` class (`labels` input to the forward method). You can choose the labels you like.

The `BertForSequenceClassification` class is **not** related to the Next Sentence Classification task used during Bert pretraining. You can use the [BertForNextSentencePrediction](https://huggingface.co/pytorch-transformers/model_doc/bert.html#bertfornextsentenceprediction) class if you want to do next sentence prediction (classification).

**Comment by ArthurCamara** [comment]
Yes, I'm sorry I meant the  [BertForNextSentencePrediction](https://huggingface.co/pytorch-transformers/model_doc/bert.html#bertfornextsentenceprediction) class.

**Comment by thomwolf** [comment]
Yes, in this case, you should follow the docstring: `0 indicates sequence B is a continuation of sequence A, 1 indicates sequence B is a random sequence.`
